tests/robot-tests/tests/libs/slack.py

```diff
@@ -71,16 +92,17 @@ def send_slack_report(env: str):
 
     print("Sent UI test statistics to #build")
 
-    client = WebClient(token=slack_bot_token)
-    print('Sending UI test report to #build')
-
-    shutil.make_archive('UI-test-report', 'zip', PATH)
-    try:
-        client.files_upload(
-            channels='#build',
-            file='UI-test-report.zip',
-            title='test-report.zip',
-        )
-    except SlackApiError as e:
-        print(f'Error uploading test report: {e}')
-    os.remove('UI-test-report.zip')
+    if should_send_report:
+        client = WebClient(token=slack_bot_token)
+        print('Sending UI test report to #build')
+
+        shutil.make_archive('UI-test-report', 'zip', PATH)
```

Comment:
I know it hasn't changed as part of this PR, but I'd rename the `PATH` variable to something like `PATH_TO_TEST_RESULTS`

Reply:
Don't really agree with this. A bit too verbose for my liking. If we had multiple directories defined in this file then this would make sense.

Reply:
I thought it was too ambiguious as it is - but I'm not precious about it - so feel free to leave it.